docs/en/basic_concepts/structures.md

```diff
@@ -1,3 +1,281 @@
+
+# Post Processor
+def get_single_prediction(
+        self,
+        probs: torch.Tensor,
+        data_sample: Optional[TextRecogDataSample] = None,
+    ) -> Tuple[Sequence[int], Sequence[float]]:
+```
+
+### Key Information Extraction Task Data Abstraction KIEDataSample
+
+[`KIEDataSample`](mmocr.structures.KIEDataSample) is used to encapsulate the data needed for the KIE task. It also agrees on two properties, `gt_instances` and `pred_instances`, which are used to store annotation information and prediction results respectively.
+
+|                |                                 |             |
+| -------------- | ------------------------------- | ----------- |
+| Field          | Type                            | Description |
+| gt_instances   | [`InstanceData`](#instancedata) | Annotation  |
+| pred_instances | [`InstanceData`](#instancedata) | Prediction  |
+
+The [`InstanceData`](#instancedata) fields that will be used by this task are shown in the following table.
+
+|             |                         |                                                                                                                                                                        |
+| ----------- | ----------------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| Field       | Type                    | Description                                                                                                                                                            |
+| bboxes      | `torch.Tensor(float32)` | Bounding boxes `[x1, x2, y1, y2]` with the shape `(N, 4)`.                                                                                                             |
+| labels      | `torch.LongTensor`      | Instance label with the shape `(N, )`. By default, MMOCR uses `0` to represent the "text" class.                                                                       |
+| texts       | `list[str]`             | The textual content of each instance with the shape `(N, )`，used for e2e text spotting or KIE task.                                                                   |
+| edge_labels | `torch.IntTensor`       | The adjacency matrix between nodes with the shape `(N, N)`. In the KIE task, the optional values for the state between nodes are `-1` (ignored, not involved in loss calculation)，`0` (disconnected) and `1`(connected). |
+| edge_scores | `torch.FloatTensor`     | The prediction confidence of each edge in the KIE task, with the shape `(N, N)`.                                                                                       |
+
+Since there is no unified standard for model implementation of KIE tasks, the design currently considers only [SDMGR](../../../configs/kie/sdmgr/README.md) model usage scenarios. Therefore, the design is subject to change as we support more KIE models.
```

Comment:
````suggestion
```{warning}
Since there is no unified standard for KIE model implementation, the design currently considers only [SDMGR](../../../configs/kie/sdmgr/README.md) model usage scenarios. Therefore, the design is subject to change as we support more KIE models.
```
````

Reply:
fix CN as well